constrain/library/AppendixGHVACSystemFanOperation.py

```diff
@@ -51,30 +61,42 @@
 ```
 
 ### Data requirements
-- o: number of occupants sensed in the zones served by the system.
-- fan_runtime_fraction: system fan runtime fraction (between 0 and 1).
-- m_oa: system outdoor air flow rate.
-- tol_o: occupancy threshold; below that value the zones are considered unoccupied.
+
+- number_occupants: Number of occupants
+  - Data Value Unit: count
+  - Data Point Affiliation: Zone occupancy
+
+- fraction_runtime_fan: Fan runtime fraction
+  - Data Value Unit: fraction
+  - Data Point Affiliation: System operation
+
+- flow_volumetric_air_outdoor: Outdoor air flow rate
+  - Data Value Unit: volumetric flow rate
+  - Data Point Affiliation: System ventilation
 
 """
 
 from constrain.checklib import RuleCheckBase
 
 
 class AppendixGHVACSystemFanOperation(RuleCheckBase):
-    points = ["o", "fan_runtime_fraction", "m_oa", "tol_o"]
+    points = [
+        "number_occupants",
+        "fraction_runtime_fan",
+        "flow_volumetric_air_outdoor",
+    ]
     potential_failures_counter = 0
```

Comment:
not used anywhere? not directly related to this PR. likely a logic issue

Reply:
```potential_failures_counter ``` is used in line 100 and 116. We can revisit this logic later.

Reply:
> `potential_failures_counter ` is used in line 100 and 116. We can revisit this logic later.

I see. I only looked at verify. did not check check_bool